Gemfile

```diff
@@ -32,6 +32,12 @@ gem "puma", "~> 2.16.0"
 
 gem 'pg', platforms: :ruby
 
+group :production, :staging do
```

Comment:
OK, I see why you're doing this. Its for authing into VACOLS. we should not do this, as not every user of Caseflow will be associated with an RO. We should just use `station_id`